src/store/actions/LoginAction.js

```diff
@@ -0,0 +1,19 @@
+import { GET_USERLOGIN} from "../types";
+import axios from 'axios';
+
+export const getLoginUserInstace = (targetuserId) => async dispatch => {
+    try {
+        const res = await axios.get(`http://localhost:8000/api/users/users/${targetuserId}/`)
```

Comment:
This one need to change to localhost first since we need to test it with BE. Will update later on if backend give us the endpoints

Reply:
Noted